label_studio/core/utils/schema/label_config_schema.json

```diff
@@ -199,11 +200,43 @@
         }
       }
     },
-	"MaybeMultipleTextAreas": {
+    "MaybeMultipleTextAreas": {
       "anyOf": [{
         "type": "array",
         "items": {"$ref": "#/definitions/TextArea"}
       }, {"$ref": "#/definitions/TextArea"}]
+    },
+    "Ranker": {
```

Comment:
are you sure we still need this? 
I have concerns when we need to extend Ranker with some new xml attributes or logic, and this scheme will stop us doing it, however, it will be bad.

Reply:
Not needed from what I can see, removing it 👍